packages/flutter/test/material/search_test.dart

```diff
@@ -1164,6 +1190,9 @@ class _TestSearchDelegate extends SearchDelegate<String> {
   final Widget? flexibleSpace;
   static const Color hintTextColor = Colors.green;
 
+  @override
+  double? get leadingWidth => 16;
```

Comment:
This should probably be made variable so all users of `_TestSearchDelegate` do not inherit this value.

Reply:
Yep, i removed this line and updated to test. Thank you